root/static/scripts/common/MB/Control/Menu.js

```diff
@@ -41,6 +41,10 @@ MB.Control.HeaderMenu = function () {
         }, self.timeout_msecs);
     });
 
+    $('.header .menu-header').click(function () {
+        $(this).siblings('ul').toggle();
```

Comment:
This behaves kinda weirdly in normal browsers, because if you click the menu-header it stays invisible even if you hover over it again. I was experimenting with different code to find something that works in the browser and my iphone, and got this:

```JavaScript
    $('.header .menu-header').on('click touchend', function (event) {
        event.preventDefault();
        event.stopPropagation();
        var ul = $(this).siblings('ul');
        $('.header ul.menu li ul').not(ul).css('left', '-10000px');
        ul.css('left', ul.css('left') === 'auto' ? '-10000px' : 'auto');
    });
```
^ can you see if that behaves how you expect too?